### ISSUE TITLE
Google Play Release

### ISSUE THREAD

**Issue by OxygenCobalt** [OxygenCobalt/Auxio#572]
Not doing this because I want to, but rather to advance my career skills. This variant of Auxio will not have any proprietary integrations.

**Comment by RustoMCSpit** [comment]
> Not doing this because I suddenly don't like F-Droid, but rather to advance my career skills.
> 
> * This variant of Auxio will not have any proprietary components, just like the F-Droid release.
> * It'll probably require me to reformat the existing screenshots in some way. Not just to be more conventionally nice-looking, but also eliminating the copyright-ambiguous album art usage. The latter might be done by using album art from an artist that has given explicit approval, or making up some fake placeholder music. Latter is easier, so I'll probably do that.
> * Deal with whatever other kinds of guidelines google expects from me.

dumb idea but you could contact small indie musicians and say youll put their songs as the images as an 'ad'

**Comment by OxygenCobalt** [comment]
> dumb idea but you could contact small indie musicians and say youll put their songs as the images as an 'ad'

Me on my way to e-mail every artist on the RateYourMusic 2023 charts to use their likeness in funny rational music app

![output](https://github.com/OxygenCobalt/Auxio/assets/65370175/e7783d10-9aae-4f6c-8166-657627b0fac5)

**Comment by OxygenCobalt** [comment]
Seems like any kind of "probably copyrighted" imagery is disallowed actually. So even indie cover art could get flagged. I think I could use unsplash stock images and made up names instead.

**Comment by rhynec** [comment]
@OxygenCobalt I know NIN released 'Ghosts' and 'The Slip' under a creative commons non-commercial license. As far as I'm aware (and wikipedia claims this too), the license extends to the artwork as well.

Additionally, King Gizzard released their album Polygondwanaland to the public domain along with its artwork.

You could also obtain some albums from the FMA, but I'm unsure if the CC licenses extend to the art. VLC has some examples on their goople play page which you could safely use.

Bonus: You could theoretically create parodies of BCNR's first album and singles since they used images from Unsplash. Or Phoenix's Alpha Zulu since the base image for the artwork is from an old painting. But probably not worth the effort, though.

**Comment by alphanerdgen** [comment]
@OxygenCobalt 

If you're going to use made-up names, then may I suggest:

Elon Zuckerberg - Cage Fight

DJ Jinping - That's All Xi Said

Phat GPT - High on Data

Pottify - Take a Dump (feat. Piratebae)

**Comment by Martysh12** [comment]
I can create some fake album covers if you decide to come back to this issue some time

**Comment by etyarews** [comment]
I understand this isn't very likely to actually contribute to anything, but as my previous request was fulfilled, I want you to mark this issue as prioritized as I think this will both help your career and Auxio's growth.

**Comment by RustoMCSpit** [comment]
admittedly this is the thing that will get auxio donations more than anything

**Comment by etyarews** [comment]
Yeah, and if @OxygenCobalt is willing to do it, he could set up some donation inside the app. I understand Google Play has a 5% increase in the fee compared to Github, however, as a customer outside the US and Europe, I find it way easier to pay using Google's payment method than using Github.

**Comment by OxygenCobalt** [comment]
Hey @etyarews, I'll prioritize this but I have zero ETA on this. I maybe only have a handful of minutes or so per day to dedicate to Auxio right now, so I'm trying to put out as many fixes as I can for 3.6.0 and 4.0.0 first.

**Comment by graphene1over** [comment]
Great news, publishing on GPlay is a good idea!

> Not doing this because I suddenly don't like F-Droid

Just to make sure, are you aware of the [security issues](https://privsec.dev/posts/android/f-droid-security-issues/) associated with fdroid?

**Comment by OxygenCobalt** [comment]
Alright, I'm aiming for a Google Play release after 4.0.0, as I'm updating Auxio's brand presence after that.

**Comment by OxygenCobalt** [comment]
I've done a ton of thought about this. In short, I don't think I'll be going ahead with this in the future.

The fact that I can barely keep up with the maintenance burden post-v4.0.0 points that I can't really maintain Auxio as a "free" endeavor if it was provided at a Google Play level scale.

I think the only circumstance Auxio lands on Google Play is if I wind up going the "premium" route. An app that's paid, like Poweramp is. I wouldn't paywall existing features (that sucks), but there is a possibility that the paid version would get out of scope features or early access to new in-scope features.

I don't know if I will go for this. I don't wish to alienate users or betray any values. Just letting you all know what I am considering.

**Comment by n-ce** [comment]
@OxygenCobalt Many FOSS apps go the premium route on PlayStore, for the exact same app that is free on github/f-droid. The premium is paid for getting playstore conveniences as well as supporting the developer.

Here are some examples : 
https://github.com/dessalines/habit-maker
https://github.com/IacobIonut01/Gallery
https://github.com/OpenTracksApp/OpenTracks

**Comment by OxygenCobalt** [comment]
Okay, that's really helpful @n-ce 

I think I'm going to consider going down this route long-term. Might require re-licensing the app to LGPL though, which in turn needs #761 so I can untangle all the contributions. Will update as I go along.

**Comment by OxygenCobalt** [comment]
I think I'm just going to go ahead w/doing a Google Play release with a Price Tag as it stands. Just a premium for faster updating and not having to use another app store, more or less. I'll see how things change in the future.
